lib/Runtime/Library/TypedArray.cpp

```diff
@@ -851,6 +851,13 @@ namespace Js
         return PropertyQueryFlags::Property_NotFound_NoProto;
     }
 
+    BOOL TypedArrayBase::DeleteItem(uint32 index, Js::PropertyOperationFlags flags)
```

Comment:
>flags [](start = 77, length = 5)

when running in strict mode, the flags will contain PropertyOperation_StrictMode
methods such as copyWithin would pass PropertyOperation_ThrowOnDeleteIfNotConfig